common/lib/xmodule/xmodule/modulestore/mixed.py

```diff
@@ -266,6 +266,21 @@ def get_items(self, course_key, **kwargs):
         return store.get_items(course_key, **kwargs)
 
     @strip_key
+    def get_course_summaries(self, **kwargs):
+        """
+        Returns a list containing the course information in CourseSummary objects.
+        Information contains `location`, `display_name`, `locator` of the courses in this modulestore.
+        """
+        course_summaries = {}
+        for store in self.modulestores:
+            for course_summary in store.get_course_summaries(**kwargs):
+                course_id = self._clean_locator_for_mapping(locator=course_summary.id)
+                if course_id not in course_summaries:
+                    # course is indeed unique. save it in result
```

Comment:
Ah, @nasthagiri , git blame leads back to you :). Were there cases where there were identical course ids across modulestores? I'm guessing it's safe to assume that course ids are unique across modulestores, but it sounds like this may have not been the case when you implemented this.

Intuitively, I don't think we'll need this check, and if we do, should we log something to tell what the duplicate course id is?